swan-lake/learn/observing-ballerina-code.md

```diff
@@ -65,70 +67,61 @@ service /hello on new http:Listener(9090) {
 
 ### Step 4 - Observing the 'Hello World' Ballerina Service
 
-Observability is disabled by default and can be enabled by using the `--b7a.observability.enabled=true` flag or updating the configurations.
+By default, Observability is not included in the executable created by Ballerina. It can be added
+by using the --observability-included build flag or by adding the following section to the `Ballerina.toml` file.
 
-When Ballerina observability is enabled, Ballerina runtime exposes internal metrics via an HTTP endpoint for metrics
-monitoring and tracers will be published to Jaeger. Prometheus should be configured to scrape metrics from
-the metrics HTTP endpoint in Ballerina.
-
-Ballerina logs are logged on to the console. Therefore, the logs need to be redirected to a file, which can then be
-pushed to [Elastic Stack](#distributed-logging) to perform the log analysis.
-
-#### Starting the Service Using a Flag
-
-The Ballerina service is observable with default settings when the `--b7a.observability.enabled=true` flag is used along with the Ballerina
-`run` command to start the service.
-This lets you collect the distributed tracing information with Jaeger and metrics information with Prometheus.
-
-```bash
-$ bal run hello_world_service.bal --b7a.observability.enabled=true
-
-[ballerina/http] started HTTP/WS listener 0.0.0.0:9797
-ballerina: started Prometheus HTTP listener 0.0.0.0:9797
-ballerina: started publishing tracers to Jaeger on localhost:5775
-[ballerina/http] started HTTP/WS listener 0.0.0.0:9090
+```toml
+[build-options]
+observabilityIncluded=true
 ```
 
-Redirect the standard output to a file if you want to monitor logs.
+To include the Prometheus and Jaeger extensions into the executable,
```

Comment:
```suggestion
To include the Prometheus and Jaeger extensions into the executable, the
```

Reply:
Fixed in 146f6434f70aee59d33f47a274e40ed2ac703d6a